src/Grpc/JsonTranscoding/test/Microsoft.AspNetCore.Grpc.JsonTranscoding.Tests/UnaryServerCallHandlerTests.cs

```diff
@@ -871,6 +871,81 @@ public async Task HandleCallAsync_RpcExceptionThrown_StatusReturned()
         Assert.Equal(debugException, exceptionWrite.Exception);
     }
 
+    [Fact]
+    public async Task HandleCallAsync_RpcExceptionThrown_StatusDetailsReturned()
```

Comment:
Duplicate this test but set `grpc-status-details-bin` on the context's `ResponseTrailers` collection. Throw a non-gRPC exception like `InvalidOperationException`.